spring/boot2-actuator-autoconfigure/build.gradle

```diff
@@ -3,8 +3,8 @@ dependencies {
     api libs.javax.inject
     compileOnly libs.javax.validation
 
-    // TODO(anuraaga): Consider removing these since this module does not have related functionality.
-    optionalApi libs.micrometer.prometheus
+    optionalApi project(':prometheus1')
```

Comment:
Do we need an optional `:prometheus1` dependency here? I know it is needed for the compiler. But it does not make sense because `prometheus1` is incompatible with Spring Boot 2. https://github.com/micrometer-metrics/micrometer/wiki/1.13-Migration-Guide#spring-boot

How about adding `:prometheus0` to avoid the compilation problem. We may keep the `prometheus0` module until we drop support for Spring Boot 2. 
I'm not confident that my idea is good. 😅

Reply:
Never mind. Let's consider adding `:prometheus0` later when we no longer use `micrometer.prometheus.legacy` in the core module.